Packs/Code42/Integrations/Code42/Code42_test.py

```diff
@@ -1205,22 +1305,65 @@ def test_client_lazily_inits_sdk(mocker):
     assert client._sdk is not None
 
 
-def test_client_when_no_alert_found_raises_exception(code42_sdk_mock):
+def test_client_when_no_alert_found_raises_alert_not_found(code42_sdk_mock):
     code42_sdk_mock.alerts.get_details.return_value = (
-        """{'type$': 'ALERT_DETAILS_RESPONSE', 'alerts': []}"""
+        json.loads('{"type$": "ALERT_DETAILS_RESPONSE", "alerts": []}')
     )
     client = create_client(code42_sdk_mock)
-    with pytest.raises(Exception):
+    with pytest.raises(Code42AlertNotFoundError):
```

Comment:
These tests were passing, but not for the reason they were intended to, so I fixed that